prow/gerrit/adapter/adapter.go

```diff
@@ -47,6 +47,7 @@ import (
 const (
 	inRepoConfigRetries = 2
 	inRepoConfigFailed  = "Unable to get inRepoConfig. This could be due to a merge conflict (please resolve them), an inRepoConfig parsing error (incorrect formatting) in the .prow directory or .prow.yaml file, or a flake. For possible flakes, try again with /test all"
+	newChangesThreshold = 50
```

Comment:
Also adding to the previous comment, the idea of a fixed arbitrary threshold without any explanation why this value is not any other different does not seem to be a good idea.